homeassistant/util/async.py

```diff
@@ -97,6 +98,10 @@ def run_coroutine_threadsafe(coro, loop):
 
     Return a concurrent.futures.Future to access the result.
     """
+    ident = getattr(loop, '_thread_ident', None)
```

Comment:
Alternatively, if it was assumed that 99% of the time, this attribute is present:

```python
try:
    ident = loop._thread_ident
except AttributeError:
    ident = None
```